index.html

```diff
@@ -1,229 +1,246 @@
-                <div class="des3">
-                <img class="h4" src="image/skills.png" alt="">
-                <ul>
-                    <li class="lastli3">Codepen</li>
-                    <li class="lastli3">Terminal</li>
-                    <li class="lastli3">codekit</li>
-                    <li class="lastli3">Github</li>
-                    <li class="lastli3">Gitlab</li>
-                </ul>
-            </div>
-             
-             </div>
-
-            </section>
-           
-
-
-
-
-
-             
-
-
-
-            </div>
+        <h3>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </h3>
+
+        <button class="btn11">Get My Resume</button>
+
+        <hr class="hr2" />
+        <div class="description">
+          <div class="des1">
+            <img src="image/lang.png" alt="" />
+            <ul class="ul44">
+              <li class="lastli">Javascript</li>
+              <li class="lastli">Ruby</li>
+              <li class="lastli">Html</li>
+              <li class="lastli">css</li>
+            </ul>
+          </div>
+
+          <div class="des2">
+            <img class="h4" src="image/frame.png" alt="" />
+            <ul>
+              <li class="lastli2">Bootstrap</li>
+              <li class="lastli2">CapyBara</li>
+              <li class="lastli2">Selenium</li>
+              <li class="lastli2">RSPec</li>
+            </ul>
+          </div>
+
+          <div class="des3">
+            <img class="h4" src="image/skills.png" alt="" />
+            <ul>
+              <li class="lastli3">Codepen</li>
+              <li class="lastli3">Terminal</li>
+              <li class="lastli3">codekit</li>
+              <li class="lastli3">Github</li>
+              <li class="lastli3">Gitlab</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <footer>
+ 
        
-      
-
-
+      >
+        <h2 class="h2footer">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in.
+        </h2>
+
+        <form class="input"  action="https://formspree.io/f/mzbokrpo"
+        method="POST">
+          
+         
+          <input
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <hr class="hrtext" />
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email address"
+            required
+          />
+          <hr class="hrtext" />
+          <textarea
+            name="textarea"
+            id="textarea"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required>
```

Comment:
- [ ] Please kindly add a placeholder text to the `textarea` input as requested by the previous reviewer